tests/SideBySide/DataTypes.cs

```diff
@@ -624,8 +624,8 @@ public void GetSchemaTable(string column, string table, int columnSize, Type dat
 					Assert.Equal(column, schema["ColumnName"]);
 					int ordinal = 2; // https://bugs.mysql.com/bug.php?id=61477
 					Assert.Equal(ordinal, schema["ColumnOrdinal"]);
-					Assert.Equal(columnSize, schema["ColumnSize"]);
 					Assert.Equal(dataType, schema["DataType"]);
+					Assert.Equal(columnSize, schema["ColumnSize"]);
```

Comment:
Appveyor runs the tests against this library, then against MySql.Data (to check for backwards compatibility). In this case, it would be OK to add a `#if BASELINE` check for the cases we know Oracle's connector has bugs (GUIDs and BLOBs).